catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -104,7 +108,11 @@ public class SolrFilterDelegate extends FilterDelegate<SolrQuery> {
           "spatialContextFactory",
           JtsSpatialContextFactory.class.getName(),
           "validationRule",
-          ValidationRule.repairConvexHull.name());
+          ValidationRule.repairConvexHull.name(),
+          "normWrapLongitude",
+          "true",
+          "allowMultiOverlap",
```

Comment:
Forgot to mention why this is necessary. Sometimes, we have polygons that share a border, such as the polygons of the Russia WKT on the antimeridian:
![Screen Shot 2022-05-23 at 1 29 47 PM](https://user-images.githubusercontent.com/4495447/169900893-eef12bfc-0520-40a9-9c27-61a1b56a1db3.jpg)
Adding a buffer causes them to overlap.